Stresser/AutoEtwTraceSession.cpp

```diff
@@ -64,7 +64,10 @@ void AutoEtwTraceSession::startTrace()
 
 	auto* properties = reinterpret_cast<PEVENT_TRACE_PROPERTIES>(this->m_propertiesBuffer.get());
 
-	//ULONG error = ::StartTrace(&this->m_traceSessionHandle, KERNEL_LOGGER_NAME, properties);
+	// If exists stop the trace session with the same name
+	EVENT_TRACE_PROPERTIES prop = { sizeof(EVENT_TRACE_PROPERTIES) };
+	ControlTrace(NULL, this->m_sessionName.c_str(), &prop, EVENT_TRACE_CONTROL_STOP);
```

Comment:
Verify return value, throw exception if needed, log message if needed,
handle locally if needed.

OR wrapped with try-catch and don't handle any thing and comment why you didn't handle anything.
Remember call to API function could led to SEH exception which could be hard to debug if we didn't log it when the exception was thrown.